Misc/NEWS.d/next/Library/2018-09-07-10-16-34.bpo-34604.xL7-kG.rst

```diff
@@ -0,0 +1,2 @@
+Fix possible mojibake in `pwd.getpwnam` and `grp.getgrnam`. Patch by William
```

Comment:
would you mind to add "... in the error message." to clarify that it doesn't affect the regular case?